test/features/admin_reporting.feature

```diff
@@ -0,0 +1,43 @@
+
+@reporting
+
+Feature: AdminReporting
+
+    Scenario: As an admin user of my organisation, I can view 'My Reports' tab in the dashboard and show the "Admin Report" with filters and table columns
```

Comment:
Scenario names should not include double quotes, as they interfere with taking screenshots.